### PR TITLE
fix: width adjusted at empty section container

### LINKED ISSUE #735: Bug - Make the my seedlots page empty state centred
Make the empty state information go to the page centre.

Image

### PR DESCRIPTION
**Pull request by ArthurEncr**
# Description
Closes #735 

#### New
- Adjusted width at empty section of seedlot table 
- Empty section icon changed
- Empty section icon color changed

#### Changed
- Nothing

#### Removed
- None

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-42-frontend.apps.silver.devops.gov.bc.ca/)
[Backend](https://nr-spar-42-backend.apps.silver.devops.gov.bc.ca/actuator/health)
[Oracle-API](https://nr-spar-42-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-42-frontend.apps.silver.devops.gov.bc.ca/)
[Backend](https://nr-spar-42-backend.apps.silver.devops.gov.bc.ca/actuator/health)
[Oracle-API](https://nr-spar-42-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

**Review by craigyu — CHANGES_REQUESTED** [review]
https://www.figma.com/file/xOTQWn4xwyIYGJQPo1OXvm/Registration-process-A-class---New-Version?type=design&node-id=6582-116390&mode=design&t=nnfnHZQ9VvHAf47D-4

Working as expected! 
but there has a been a design change to the icon/pictogram of the empty section, should be a quick update

**Review by mgaseta — APPROVED** [review]
Other than Craig's markup, LGTM!

**Review by RMCampos — APPROVED** [review]
LGTM 🚀

**Review by craigyu — CHANGES_REQUESTED** [review]
image

Did you forget to push your changes by any chance? let's add that hot coffee mug ☕️

**Comment by ArthurEncr** [comment]
> image
> 
> Did you forget to push your changes by any chance? let's add that hot coffee mug ☕️

A hot coffee for Craig (I guess I forgot then, I just made another push)

File: frontend/src/components/EmptySection/styles.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
   width: 3rem !important;
   height: 3rem !important;
   padding: 0 !important;
-  color: colors.$gray-70;
+  color: colors.$blue-60;
```
Comment (craigyu) [inline frontend/src/components/EmptySection/styles.scss:7]: Please change the colour of this icon only for that page, changing it for this component will affect other pages that use it, such as:

image

You can probably do that by adding a color override in  `frontend/src/components/SeedlotTable/styles.scss` under `.empty-recent-seedlots`

**Review by craigyu — CHANGES_REQUESTED** [review]

File: frontend/src/components/EmptySection/styles.scss (hunk 1 above)
Comment (craigyu) [reply frontend/src/components/EmptySection/styles.scss:7]: @mmarsoleta should we change the colour for empty icon/pictogram for all empty sections or just for the empty seedlot table?

Comment (mmarsoleta) [reply frontend/src/components/EmptySection/styles.scss:7]: For all the empty states' icons, we will be using the blue color for now on.  Thank you for asking!

**Pull request merged**